index.html

```diff
@@ -1,12 +1,45 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    
+    <section class="container">
+        <nav class="mobile-nav">
+            <div class="shapes">
+                <div class="images">
+                    <img id="triangleImage" src="images/ShapeTriangle.png" alt="triangel shape">
+                    <img id="rectangleImage" src="images/Shape.png" alt="rectangle shape">
+
+                </div>
+            </div>
+            <div class="hamburger">
+                <span class="rec rectangle1"></span>
+                <span class="rec rectangle2"></span>
+                <span class="rec rectangle3"></span>
+            </div>
+        </nav>
+        <div class="main">
+            <h1 id="header">Ayesha Arshad</h1>
+            <p id="discription">Hello! I am a software developer! I can help you build a product, feature or
+                website. Taka a look of my works.
+                If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+            <div class="primary-button">
+                <button id="text" type="button">
+                    Start collaboration</button>
+            </div>
+            <div class="bg-image">
+                <img src="images/bg-cover-page.jpg" alt="background image for cover page">
```

Comment:
- [ ] as the previous reviewer asked for this illustration is only for design  it doesn't make sense to use it as a `<img>` tag so Encourage to use it as a background image using CSS